src/Documents.ts

```diff
@@ -125,8 +126,6 @@ export class Class implements Pointer.Library, Pointer.Validator, ObjectSchema.R
 		return this.pointers.get( localID );
 	}
 
-	removePointer( id:Pointer.Class ):boolean;
-	removePointer( id:string ):boolean;
```

Comment:
Even though removing this doesn't actually make a runtime difference, it has an impact on developers using the SDK. If they have the two separate signatures, passing a string would actually make some IDEs do something like this: `removePointer( [id]: "someID" )`, whereas with just one signature it would end up looking like: `removePointer( [idOrPointer]: "someID" )`. Please add the signatures back but change the one that accepts the pointer to: `removePointer( pointer:Pointer.Class ):boolean;`

Reply:
I see your point with the name, but as referred in the best practices of TypeScript: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#use-union-types

Having two different signatures where both has a single parameter is problematic for the user, because is common to have:
```typescript
const theParameter:string | Pointer.Class = /* ... */
carbon.documents.removePointer( theParameter ); // Typescript throws an Error
```
And TypeScript can't infer which signature to use.

Reply:
I think that case is going to be more rare than the one I described but if it's a TypeScript best practice we'll follow it 👍